airflow/utils/state.py

```diff
@@ -58,6 +58,13 @@ class TaskInstanceState(str, Enum):
     SKIPPED = "skipped"  # Skipped by branching or some other mechanism
     DEFERRED = "deferred"  # Deferrable operator waiting on a trigger
 
+    # Not used anymore, kept for compatibility.
```

Comment:
According to https://github.com/apache/airflow/pull/33746#issuecomment-1698956131, should we add a comment to remind us to remove it on the next major release?

Reply:
Not _that_ useful since I’d assume the cleanup process would involve searching for `:meta private:` anyway. But a comment is easy and would not hurt.